Expensify/App
# Remove the reviewer checklist from the PR template

### Fixed Issues
$ https://github.com/Expensify/Expensify/issues/239867


### Tests
None. Can only be tested once merged

### Offline tests
None - this doesn't affect the app at all

### QA Steps
1. Create a new PR for any simple file change
2. Verify that the template no longer includes the reviewer checklist in the description
3. Close out the PR

### PR Author Checklist
Checklist and screenshots do not apply since this is not a change to the app code
